Solutions/Azure Cloud NGFW by Palo Alto Networks/Analytic Rules/CloudNGFW-NetworkBeaconing.yaml

```diff
@@ -0,0 +1,60 @@
+id: f0be259a-34ac-4946-aa15-ca2b115d5feb
+name: Palo Alto - potential beaconing detected
+description: |
+  'Identifies beaconing patterns from Palo Alto Network traffic logs based on recurrent timedelta patterns.
+  The query leverages various KQL functions to calculate time deltas and then compares it with total events observed in a day to find percentage of beaconing.
+  This outbound beaconing pattern to untrusted public networks should be investigated for any malware callbacks or data exfiltration attempts.
+  Reference Blog:
+  http://www.austintaylor.io/detect/beaconing/intrusion/detection/system/command/control/flare/elastic/stack/2017/06/10/detect-beaconing-with-flare-elasticsearch-and-intrusion-detection-systems/
+  https://techcommunity.microsoft.com/t5/microsoft-sentinel-blog/detect-network-beaconing-via-intra-request-time-delta-patterns/ba-p/779586'
+severity: Low
+status: Available
+requiredDataConnectors:
+  - connectorId: CloudNgfwByPAN

```

Comment:
Hi @davidokeyode, there is no data connector with such ID, either add data connectors with this ID or change the ID here with the existing data connector (depending on the logic) and this includes for all the three analytical rules

Reply:
@v-rbajaj - We defined a data connector with that name here: https://github.com/davidokeyode/Azure-Sentinel/blob/palo-alto-networks-azure-cloudngfw/Solutions/Azure%20Cloud%20NGFW%20by%20Palo%20Alto%20Networks/Data%20Connectors/CloudNgfwByPAN.json

Do we need to define it somewhere else?